lib/api/client-commands/alerts/setText.js

```diff
@@ -27,12 +28,31 @@ const ProtocolAction = require('../_base-action.js');
  * @link /#send-alert-text
  * @api protocol.userprompts
  */
-module.exports = class Session extends ProtocolAction {
+class SetAlertText extends ClientCommand {
   static get isTraceable() {
     return true;
   }
 
+  performAction(callback) {
+    const {alertText} = this;
+
+    this.transportActions
+      .setAlertText(alertText, callback)
+      .catch(err => {
+        Logger.error(err);
```

Comment:
do we need the logger here? isn't it already logged by the transport actions?

Reply:
There are some errors which aren't reported: https://github.com/nightwatchjs/nightwatch/pull/3610#discussion_r1132081946

Reply:
Yeah that's true, type and syntax errors won't be reported by the error handler in the transport actions. But I think for transport/selenium errors, we'd log the errors twice.

Reply:
Nope, transport/selenium errors are handled elsewhere so they don't appear in the `catch` clause above. But without the above `catch` clause, the type and syntax error remain unhandled and appear as below:
![image](https://user-images.githubusercontent.com/39924567/232020193-ee66fff5-c654-4784-9b87-2d292cf7f5f4.png)

One thing we can do is instead of having the `catch` clause in every other command, we can put just one `catch` clause over [here](https://github.com/nightwatchjs/nightwatch/blob/2bb54d36a07d6bb4b295ae84b73111a6c58aa12c/lib/core/client.js#L313)?

Reply:
yeah, that sounds good.

Reply:
Done